crates/core/src/db/models.rs

```diff
@@ -226,3 +211,81 @@ pub struct ListingsTripleJoinRow {
     /// Listing has already been sold once
     pub primary_sale_happened: bool,
 }
+
+/// A row in the `metadata_jsons` table
+#[derive(Debug, Clone, Queryable, Insertable, AsChangeset)]
+pub struct MetadataJson<'a> {
+    // Table 'metadata_jsons'
+    /// Metadata Address
+    pub metadata_address: Cow<'a, str>,
+    /// Metadata URI fingerprint - Cid for Ipfs and ArTxid for Arweave
+    pub fingerprint: Option<Cow<'a, Vec<u8>>>,
+    /// Metadata description
+    pub description: Option<Cow<'a, str>>,
+    /// Metadata Image url
+    pub image: Option<Cow<'a, str>>,
+    /// Metadata Animation url
+    pub animation_url: Option<String>,
+    /// Metadata External Url
+    pub external_url: Option<String>,
+    /// Metadata Category
+    pub category: Option<String>,
+    /// Metadata Timestamp
+    pub updated_at: Option<NaiveDateTime>,
+    /// Metadata URI raw json
+    pub raw_content: Option<Cow<'a, serde_json::Value>>,
+}
```

Comment:
If you get a large type mismatch from trying to select columns check to make sure that this model has the columns defined in the same order as the schema.rs and the types are the same. In this case the null values needed tom additional options and reorder. @imabdulbasit